pages/settings/index.vue

```diff
@@ -0,0 +1,333 @@
+<template>
+  <v-container>
+    <v-row>
+      <v-col cols="12" offset-lg="1" md="5" lg="4" class="pt-0 pb-4">
+        <v-card
+          v-if="!isEmpty(systemInfo) && $auth.user.Policy.IsAdministrator"
+          :class="{ 'mb-4': !$vuetify.breakpoint.mobile }"
+        >
+          <v-card-text>
+            <v-row>
+              <v-col>
+                <v-row>
+                  <v-col cols="5" class="pt-0 pb-0">
+                    <span>{{ $t('server') }}</span>
+                  </v-col>
+                  <v-col class="pt-0 pb-0">
+                    <span>{{ systemInfo.ServerName }}</span>
+                  </v-col>
+                </v-row>
+                <v-row>
+                  <v-col cols="5" class="pt-0 pb-0">
+                    <span>{{ $t('version') }}</span>
+                  </v-col>
+                  <v-col class="pt-0 pb-0">
+                    <span>{{ systemInfo.Version }}</span>
+                  </v-col>
+                </v-row>
+                <v-row>
+                  <v-col cols="5" class="pt-0 pb-0">
+                    <span>{{ $t('operatingSystem') }}</span>
+                  </v-col>
+                  <v-col class="pt-0 pb-0">
+                    <span>{{ systemInfo.OperatingSystemDisplayName }}</span>
+                  </v-col>
+                </v-row>
+                <v-row>
+                  <v-col cols="5" class="pt-0 pb-0">
+                    <span>{{ $t('architecture') }}</span>
+                  </v-col>
+                  <v-col class="pt-0 pb-0">
+                    <span>{{ systemInfo.SystemArchitecture }}</span>
+                  </v-col>
+                </v-row>
+                <v-row>
+                  <v-col cols="5" class="py-0">
+                    <span>{{ $t('vueClientVersion') }}</span>
+                  </v-col>
+                  <v-col class="py-0">
+                    <span>{{ vueVersion }}</span>
+                  </v-col>
+                </v-row>
+              </v-col>
+              <v-col cols="3" class="d-flex justify-end">
+                <v-avatar tile size="96" color="grey">
+                  <v-icon>mdi-server</v-icon>
+                </v-avatar>
+              </v-col>
+            </v-row>
+          </v-card-text>
+        </v-card>
+        <v-card v-if="!$vuetify.breakpoint.mobile">
+          <v-card-text>
+            <v-row>
+              <v-col>
+                <p class="mb-0">
+                  <span
```

Comment:
Why in v-html and not inside the tag? Just like you do in the system version strings. Do we need to keep formatting or something?

This can lead to XSS attacks.

Reply:
The issue is that we need to translate the link and the text around it. The place of the linked word can change depending on the language, though, which means we need to translate the entire thing, link included.

Of course, the issue is that this opens us to potential issues, but I don't really see another way to do this :/

Reply:
@mrtimscampi Didn't think about the links, good point.

Not sure if it's worth it, given that no data should be inputted by any users anywhere (but I'm not by any means an XSS expert) but I just had the following idea:

Create a new component that handles this exact situation. We pass it the translation's key string and the component, based on separators, can determine what's a link and what not in the value. For instance:

"Check \[Jellyfin\]\(jellyfin.org\) docs for more information"

With a computed property, we generate two arrays, one to be used with a for that generates <span> tags and another one for <a>. The <span> tags will be generated from the array \["Check", "docs for more information"\] and the <a> from \["\[Jellyfin\]\(jellyfin.org\)"\].

We should be able to cover all the possible cases with a component that parses texts like this while keeping Vetur's complaints away.

Reply:
Do note that there is sanitation performed on the entire translated chain, using [dompurify](https://github.com/cure53/DOMPurify).

It's explicitly for these kind of cases that I added it to the source.

Vetur's warning is just that: a warning to let you know you need to sanitize whatever you pass to it, which we do already.

Reply:
@mrtimscampi Fair enough.

The only remnant is the question about the images location 😄